demos/v1/emojivoto/spec.yaml

```diff
@@ -0,0 +1,15 @@
+applicationCreator: linkerd
+type: DEMO
+logoUrl: https://avatars2.githubusercontent.com/u/10824329?s=400&v=4
+longDescription: |
+  The canonical testing application for linkerd.
+name: emojivoto
+shortDescription: |
+  The canonical testing application for linkerd.
+versions:
+  - manifestsArchive:
+      uri: https://storage.googleapis.com/sm-marketplace-registry/emojivoto.tar.gz
+    flavors:
+      - name: default
+        description: "Install emojivoto"
+    version: "latest"
```

Comment:
can you change this to v8 or similar? that's the version on the images in the manifest